hooks/environment

```diff
@@ -66,12 +66,14 @@ else
   check_command jq
   secret=$(retry vault kv get -format=json "${BUILDKITE_PLUGIN_VAULT_SECRETS_PATH}" | jq -c .data)
 fi
-
+echo "${secret}" | buildkite-agent redactor add
 if [ -n "${BUILDKITE_PLUGIN_VAULT_SECRETS_ENV_VAR}" ]; then
   eval "${BUILDKITE_PLUGIN_VAULT_SECRETS_ENV_VAR}='${secret}'"
   export "${BUILDKITE_PLUGIN_VAULT_SECRETS_ENV_VAR?}"
+  echo "${BUILDKITE_PLUGIN_VAULT_SECRETS_ENV_VAR}" | buildkite-agent redactor add
 else
   variable_name=$(generate_variable_name "$BUILDKITE_PLUGIN_VAULT_SECRETS_PATH" "$BUILDKITE_PLUGIN_VAULT_SECRETS_FIELD" "$BUILDKITE_PLUGIN_VAULT_SECRETS_PATH_DEPTH")
   eval "${variable_name}='${secret}'"
   export "${variable_name?}"
+  echo "${variable_name}" | buildkite-agent redactor add
```

Comment:
Shouldn't be needed, as this is a repeat of echoing `secret`
```suggestion
```

Reply:
Ok, so we agree we are not redacting names.

Furthermore it was either this or that. Using variable name to redact the value makes little difference.
However I used the variable_bane since it was the latest occurence to make sure that nothing could change in the middle.
In othher words, I did it for the same reason -which is not totally clear to me- that we export the variable name and not the secret itself.